test/run/ch-run_misc.bats

```diff
@@ -2,20 +2,20 @@ load ../common
 
 @test 'relative path to image' {  # issue #6
     scope quick
-    DIRNAME=$(dirname "$CHTEST_IMG")
-    BASEDIR=$(basename "$CHTEST_IMG")
-    cd "$DIRNAME" && ch-run "$BASEDIR" -- true
+    dir=$(dirname "$ch_imgtest")
```

Comment:
Perhaps `dirname_` and `basename_`? `/bin/dir` is also a command that you don't want to shadow. Or, you might simply eliminate the variables.